packages/core/onLongPress/index.test.ts

```diff
@@ -199,6 +199,7 @@ describe('onLongPress', () => {
       it('should remove event listeners after being stopped', () => stopEventListeners(isRef))
       it('should trigger longpress if pointer is moved', () => triggerCallbackWithThreshold(isRef))
       it('should trigger onMouseUp when pointer is released', () => triggerOnMouseUp(isRef))
+      it('should trigger longpress after options.delay ms when options.delay is a function', () => triggerCallbackWithDelay(isRef, () => 1000))
```

Comment:
this should set a value other than the default (e.g. `2000`).

otherwise, this test can fall through to the default delay and give us a false positive.

in fact, if you change this to 2000, the test will fail. because the helper (`triggerCallbackWithDelay`) is hard coded to wait 1000ms.

you should update that to basically do this:

```ts
await vi.advanceTimersByTimeAsync(delayFunc ? delayFunc(pointerdownEvent) : 1000)
```